core/InformationModel/Message/README.md

```diff
@@ -0,0 +1,113 @@
+# ids:Message
+
+Core `ids:Message` class with it's properties, which are equal for all messages.
+ For communication in the IDS, use the specific message types.
+
+
+## Properties of ids:Message
+
+### Table of Message Properties
+
+|**Property**|mandantory|cardinality|
+|:---|:---:|:---|
+|**[ids:correlationMessage](#idscorrelationmessage)**|x|0..1|                        
+|**[ids:securityToken](#idssecuritytoken)**|x|1|                        
+|||
+
+---
+
+### ids:correlationMessage
+
+Correlated message. Usually needed, if a messages responds to a previous
+
+> TODO: MessageProcessedNotification >>> MessageProcessedNotification?
+
+ message. A [Connector](../../Connector/) may, e.g., send a [MessageProcessedNotification](#idsmessageprocessednotificationmessage)` as
+ a response to an incoming message and therefore needs this property to
+ refer to the incoming message.
+
+
+### ids:securityToken
+
+The **mandantory** property `ìds: securityToken` contains (value) a token,
+ representing a claim, that the sender supports a certain security profile.
+
+This token is called [Dynamic Attribute Token (DAT)](../../DAPS/README.md#dynamic-attribute-token-dat),
+ (see further explanations there), so it is an instance of `ids:DynamicAttributeToken`
+ and will be provided by [Dynamic Attribute Provisioning Service (DAPS)](../../DAPS/README.md). 
+
+---
+
+## Message Types
+
+The table contans descriptions for each message, information about the payload
+ as well as changes in the properties, i.e., new properties and additional
+ mandatory properties. "Message Class" and "Message Subclass" are just for
+ structural / hierarchical descriptions. The "Message Name" contains the
+ actual Message which is used.
+ 
+New properties, which are mandatory are marked with an (*). Existing
+ properties, which are mandatory for a specific messages are listed
+ in the corresponding column.
+
+All mandatory property declarations of the core ids:Message above still hold.
+ Optional properties are supported but not required.
+
+
+### Table of Message Types
+
+| **Message Class** | Sub Class | Message Name |
+|:---|:---|:---|
+| **[Request Messages](#request-messages)**            |                                 | [ids:RequestMessage](#idsrequestmessage)
```

Comment:
List also provides ids:DescriptionRequestMessage, missing here?

Reply:
DRAFT-branch!!!! Those days I'd tried to get in touch with the ids:Messages - but noc one from IDS-IM was able to help me to lift it to a better level... maybe nowadays?!?